src/main/kotlin/sliv/tool/importer/view/ChoosingDirectoryView.kt

```diff
@@ -30,8 +31,12 @@ class ChoosingDirectoryView : Fragment() {
         }
 
         path.onChange {
-            rootTree = ProjectParser.createTreeWithFiles(it, rootTree)
+            var project = parseDirectory(it.toString())
+            rootTree = ProjectParser.createTreeWithFiles(project, rootTree)
+        }
+
+        treeview(rootTree) {
+            visibleWhen { path.isNotEmpty }
```

Comment:
What if directory by path is not valid? Empty tree view is shown, but it is better to show some placeholder